deploy/docker/Dockerfile-cypress

```diff
@@ -0,0 +1,48 @@
+# don't forget to pass environmet variables to docker run like this:
+# podman run -e CYPRESS_BASE_URL=<kiali_url> -e CYPRESS_PASSWD=<passwd> docker-cypress:latest
+# more details here https://github.com/kiali/kiali/tree/master/frontend/cypress
+
+# using same baseimage of node as its defined in package.json
+FROM cypress/base:14.17.0
+
+# Run as non-root user for MP+/OpenShift
+USER node
+
+ENV KIALI_HOME=/opt/kiali \
+    PATH=$KIALI_HOME:$PATH
+
+WORKDIR $KIALI_HOME
+
+# "Fake" dbus address to prevent errors. (is this still needed?)
+# https://github.com/SeleniumHQ/docker-selenium/issues/87
+ENV DBUS_SESSION_BUS_ADDRESS=/dev/null
+
+# A few environment variables to make NPM installs easier.
+# Good colors for most applications.
+ENV TERM xterm
+# Avoid million NPM install messages.
+ENV npm_config_loglevel warn
+# Allow installing when the main user is root.
+ENV npm_config_unsafe_perm true
+
+# copy frontend folder to container
+COPY --chown=node:node frontend/ $KIALI_HOME/
+# Install Cypress dependencies.
+RUN /usr/local/bin/yarn install --frozen-lockfile
+
+# Run Cypress tests.
+CMD ["yarn", "cypress:run:junit"]
```

Comment:
We will also need 'yarn cypress:combine:reports' I guess

Reply:
I can add `cypress:combine:reports` into the npm scripts or extend this CMD
but
I'm still not sure how do I get XML file outside of container

Reply:
We don't have to worry about that now. We just need to put the xml to correct location and it will be used later in the pipeline. The ${SHARED_DIR} should be correct location and it should be available for later pipeline steps.